sktime/markov/msm/maximum_likelihood_msm.py

```diff
@@ -14,22 +14,24 @@
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
-from typing import Optional
+from typing import Optional, Union, List
 
 import numpy as np
 from msmtools import estimation as msmest
 from scipy.sparse import issparse
 
-from .markov_state_model import MarkovStateModel
+from .markov_state_model import MarkovStateModelCollection
 from .._base import _MSMBaseEstimator
-from ..transition_counting import TransitionCountModel
+from ..transition_counting import TransitionCountModel, TransitionCountEstimator
 
 __all__ = ['MaximumLikelihoodMSM']
 
 
 class MaximumLikelihoodMSM(_MSMBaseEstimator):
     r"""Maximum likelihood estimator for MSMs (:class:`MarkovStateModel <sktime.markov.msm.MarkovStateModel>`)
-    given discrete trajectory statistics.
+    given discrete trajectory statistics. While at it's core this estimator produces instances of MSMs, it actually
```

Comment:
- its (no apostrophe)
- i feel like it might be confusing for the user to understand how to differentiate "at its core" from "actually". can this be clearer?